File: 10challenge/main.cpp
#include <iostream>

int main() {
    double width, height, area, paintNeeded;
    std::cout << "Enter the width of the wall: ";
    std::cin >> width;
    std::cout << "Enter the height of the wall: ";
    std::cin >> height;

    area = width * height;
    paintNeeded = area / 2;

    std::cout << "The area to be painted is: " << area << " square meters" << std::endl;
    std::cout << "The amount of paint needed is: " << paintNeeded << " liters" << std::endl;
    return 0;
}

File: 10challenge/main.go
package main

import (
	"fmt"
)

func main() {
	var width, height, area, paintNeeded float64
	fmt.Print("Enter the width of the wall: ")
	fmt.Scan(&width)
	fmt.Print("Enter the height of the wall: ")
	fmt.Scan(&height)

	area = width * height
	paintNeeded = area / 2

	fmt.Printf("The area to be painted is: %.2f square meters\n", area)
	fmt.Printf("The amount of paint needed is: %.2f liters", paintNeeded)
}
